rust/hello-world/config/heft.json

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Since this is a complete override, you'll need to define the clean eventAction yourself.

Reply:
Or add an `extends` clause.